docs/openstack.md

```diff
@@ -44,3 +44,8 @@ and `kube_pods_subnet` (default `10.233.64.0/18`.)
     neutron port-update e5ae2045-a1e1-4e99-9aac-4353889449a7 --allowed_address_pairs list=true type=dict ip_address=10.233.0.0/18 ip_address=10.233.64.0/18
 
 Now you can finally run the playbook.
+
+One more thing you need to is to enable IP protocol v4 (`ipip`) traffic in your security group.
```

Comment:
This is an either-or situation. Either allow the address pairs or use IPIP. By default, IPIP is not blocked in most installations.

Reply:
I can change that if you want. I personally didn't get it to work with the allowed ip address only.
Allowing explicitly that type of traffic via security groups was the solution. 
Either way, I believe the current documentation needs reviewing.